### PR TITLE
LOOP-3820 Mitigation for CGM Sensitivity analysis

### PR DESCRIPTION
**Pull request by ps2**
https://tidepool.atlassian.net/browse/LOOP-3820

### CODE REVIEW

File: LoopKit/TherapySettings.swift [hunk 1]
```diff
@@ -65,50 +65,6 @@ public struct TherapySettings: Equatable {
     }
 }
 
-extension TherapySettings: RawRepresentable {
```
Comment (ps2) [inline LoopKit/TherapySettings.swift:68]: This doesn't appear to be used in any code.

**Review by darinkrauss — APPROVED** [review]
LGTM!

File: LoopKit/LoopMath.swift [hunk 2]
```diff
@@ -208,6 +208,29 @@ extension BidirectionalCollection where Element == GlucoseEffect {
...
         return sums.reversed()
     }
+
+    /// Sums all effects into a single effect
+    ///
+    /// Requires the receiver to be sorted chronologically by endDate
+    ///
+    /// - Parameter duration: The duration of each resulting summed element
+    /// - Returns: A single GlucoseChange representing the net effect
+    public func netEffect() -> GlucoseChange? {
```
Comment (rickpasetto) [inline LoopKit/LoopMath.swift:217]: Unit test?

File: LoopKit/LoopMath.swift [hunk 3]
```diff
@@ -208,6 +208,29 @@ extension BidirectionalCollection where Element == GlucoseEffect {
 
         return sums.reversed()
     }
+
+    /// Sums all effects into a single effect
+    ///
+    /// Requires the receiver to be sorted chronologically by endDate
+    ///
+    /// - Parameter duration: The duration of each resulting summed element
```
Comment (rickpasetto) [inline LoopKit/LoopMath.swift]: [nit] `duration` isn't a parameter...?

**Review by rickpasetto — APPROVED** [review]
A nit, a request for a unit test, other than that LGTM

**Pull request merged**